addons/product/data/product_demo.xml

```diff
@@ -550,16 +556,22 @@
             <field name="weight">0.01</field>
             <field name="uom_id" ref="uom.product_uom_unit"/>
             <field name="uom_po_id" ref="uom.product_uom_unit"/>
-            <field name="default_code">FURN_6666</field>
             <field name="image_1920" type="base64" file="product/static/img/product_product_25-image.png"/>
```

Comment:
I preferred to keep the name for the git blame and for consistency since `product_product_11_product_template` has this name: "product/static/img/product_product_11-image.png". Should I rename the 2 images?